### PR TITLE
added deletion of the success message in the saveGif function

### LINKED ISSUE #6084: Deleting a 'Done. Downloading your gif!🌸' paragraph after a while
### Increasing Access

Unsure

### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [X] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### Feature enhancement details

Hi,
this is a really minor improvement. Currently, when a user successfully generated a GIF with saveGif function, p5 shows a message **Done. Downloading your gif!🌸**, which stays on the screen forever. It would be nice to remove it after some amount of time. 

`p.html('Done. Downloading your gif!🌸');`
 `setTimeout(()=> p.remove(),5000);`

Thanks!

### PR DESCRIPTION
**Pull request by agrshch**
Resolves #6084 

 Changes:

Success message after saving a gif with the saveGif() function will not last forever and will disappear in 5 seconds

#### PR Checklist

- [x] `npm run lint` passes

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by davepagurek** [comment]
@jesi-rgb how would you feel about a little X button at the end of the message to click to dismiss?

**Comment by jesi-rgb** [comment]
@davepagurek that would probably address all problems! feels like the best compromise between the two.

this also begs the question of whether the pop up is necessary altogether. i believe this little pieces of user feedback are always useful, something that was very much lacking in ccapture and other libraries, for example. they had messages like "be patient, the gif is processing" but that is clearly was not as useful. that's why i added all this little messages!

**Comment by agrshch** [comment]
Hi @jesi-rgb @davepagurek ,
first of all — the function is amazing! It works super-nice and those messages are cute :)

Currently, I am making a tool for a client. That tool makes graphics, allows users to adjust it and to export the result in several different formats. In this case, a user can make something, then export a gif, look at it, tweak something in settings, export again and so on. It's not really handy to have those notifications on top constantly, and I am afraid it would be too annoying to close them manually each time.

I think my proposal to just delete them automatically 5 seconds after is raw. Instead, those notifications could be adjustable or even optional through the 'settings' object. Also, I've been thinking about a custom callback, that could be passed through the settings object, but this is a separate theme

**Comment by agrshch** [comment]
@jesi-rgb ok, I will try to implement this then. 
Btw, do you know why the testing failed? I run the test locally and everything was fine

**Comment by jesi-rgb** [comment]
thanks so much!! @agrshch please hit us up for any questions :)

about the tests... yeah, they are very finicky. apparently there's an error with gifs not animating properly (you can see that in the build details) but that, i believe, has nothing to do with our side of gif, but rather the function that can import gif to display as images. these two are completely unrelated in the code.

we'll see about that! do not worry too much for now :P

**Comment by jesi-rgb** [comment]
thanks so much!! @agrshch please hit us up for any questions :)

about the tests... yeah, they are very finicky. apparently there's an error with gifs not animating properly (you can see that in the build details) but that, i believe, has nothing to do with our side of gif, but rather the function that can import gif to display as images. these two are completely unrelated in the code.

we'll see about that! do not worry too much for now :P

**Comment by davepagurek** [comment]
@agrshch that's a flaky test case, I'll rerun your tests if I see that one failing again. In the mean time I'll try making a fix in another PR for it :')

**Comment by agrshch** [comment]
@jesi-rgb, thank you!

**Review by davepagurek — COMMENTED** [review]
Thanks for making these changes, looks great! My only comments are about matching the code style a little more, but as far as the logic and functionality goes, it looks good to me!

File: src/image/loading_displaying.js [hunk 1]
```diff
@@ -462,7 +492,12 @@ p5.prototype.saveGif = async function(
   this._recording = false;
   this.loop();
 
-  p.html('Done. Downloading your gif!🌸');
+  if (!silent){
```
[the remark this answers is not in the document]
Comment (agrshch) [reply src/image/loading_displaying.js:495]: Thanks for noticing! Fixed that

**Review by davepagurek — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**